src/main/frontend/components/SitesIndex.js

```diff
@@ -34,6 +35,7 @@ const SitesIndex = props => {
 
   return (
     <div>
+      <p>this is site index</p>
```

Comment:
I might go with something different for the title of this page.  Index/Show/New may mean alot to us, but it won't mean much to the users visiting your website.

Reply:
Great note!  We meant to update this and just have.  Thank you